bootstrap/azure/util.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
+
+# Use the config file specified in $APOLLO_CONFIG_FILE, or default to
+# config-default.sh.
+function set_vpn() {
+  while true; do
+  read -p "Do you want to start the VPN and setup a connection now (y/n)?" yn
+    case $yn in
+        [Yy]* ) ovpn_start;ovpn_client_config; break;;
+        [Nn]* ) exit;;
+        * ) echo "Please answer y or n.";;
+    esac
+  done
+}
+
+ansible_ssh_config() {
```

Comment:
The code should be move to a common library because it is a copy of existing one which we use for aws